src/main/java/com/yahoo/bullet/querying/operations/ProjectionOperations.java

```diff
@@ -45,23 +43,12 @@ public static BulletRecord project(BulletRecord record, Projection projection, M
             String field = e.getKey();
             String newName = e.getValue();
             try {
-                copyInto(projected, newName, record, field);
-            } catch (ClassCastException cce) {
-                log.warn("Skipping copying {} as {} as it is not a field that can be extracted", field, newName);
+                if (field != null) {
+                    projected.forceSet(newName, record.extractField(field));
+                }
+            } catch (RuntimeException ignored) {
```

Comment:
No exception will be thrown.

Reply:
We don't know because forceSet calls set which may or may not support unchecked sets